src/uct/sm/mm/mm_iface.c

```diff
@@ -524,14 +531,15 @@ static UCS_CLASS_INIT_FUNC(uct_mm_iface_t, uct_md_h md, uct_worker_h worker,
         }
     }
 
+    self->dummy_fifo_ctl = ucs_malloc(sizeof(*(self->dummy_fifo_ctl)), "dummy fifo ctl");
```

Comment:
no need for malloc - can be directly inside the iface
(also - return value from malloc should be checked)